### PR TITLE
[#264] Add credential type to unable to login message

### PR DESCRIPTION
**Pull request by JessikaCastellano**
Solves [#264](https://github.com/UffizziCloud/uffizzi_app/issues/264)

### CODE REVIEW

File: core/app/models/uffizzi_core/credential.rb [hunk 1]
```diff
@@ -2,4 +2,16 @@
 
 class UffizziCore::Credential < UffizziCore::ApplicationRecord
   include UffizziCore::Concerns::Models::Credential
+  extend Enumerize
+
+  enumerize :type,
```
Comment (zipofar) [inline core/app/models/uffizzi_core/credential.rb:7]: Please check it with platform.
You can copy `core` from uffizzi_app to uffizzi_platform and update Gemfile like this:
```
uffizzi_core path: './core'
```

File: core/app/models/uffizzi_core/credential.rb [hunk 2]
```diff
@@ -2,4 +2,16 @@
...
+            in: [
+              UffizziCore::Credential::Amazon.name,
+              UffizziCore::Credential::Azure.name,
+              UffizziCore::Credential::DockerHub.name,
+              UffizziCore::Credential::DockerRegistry.name,
+              UffizziCore::Credential::Github.name,
```
Comment (zipofar) [inline core/app/models/uffizzi_core/credential.rb:13]: I think we could not use Github credentials in core

Comment (JessikaCastellano) [reply core/app/models/uffizzi_core/credential.rb:13]: The thing is that there are test that create github credentials, if I don't add this those test fails!

This one for example `core/test/controllers/uffizzi_core/api/cli/v1/projects/deployments_controller_test.rb`

```
@credential = create(:credential, :github, :active, account: @account, provider_ref: generate(:number))
```

File: core/app/models/uffizzi_core/credential.rb (hunk 1 above)
Comment (JessikaCastellano) [reply core/app/models/uffizzi_core/credential.rb]: Apply some changes on platform [PR #293](https://github.com/UffizziCloud/uffizzi_platform/pull/293)

File: core/app/models/uffizzi_core/credential.rb (hunk 2 above)
Comment (JessikaCastellano) [reply core/app/models/uffizzi_core/credential.rb]: Github credentials removed and test updated

File: core/config/locales/en.activerecord.yml [hunk 3]
```diff
@@ -13,14 +13,14 @@ en:
             password:
               password_blank: Password/Access Token can't be blank
             username:
-              incorrect: We were unable to log you in to docker registry. Please verify that your username and password are correct.
+              incorrect: "We were unable to log you in to %{type}. Please verify that your username and password are correct."
```
Comment (moklidia) [inline core/config/locales/en.activerecord.yml:16]: Let's write `into` instead of `in to`

**Pull request merged**